lib/smart_proxy_pulp_plugin/pulp_disk.rb

```diff
@@ -0,0 +1,21 @@
+module PulpProxy
+  class PulpDisk
+    def initialize(path = PulpProxy::Plugin.settings.pulp_dir)
+      @path = path
+    end
+
+    def df_h
+      `df -h #{@path}`.split("\n").last
```

Comment:
Additional note, I'd rather return bytes instead of human readable things, we can converet this in Rails during presentation.